sockeye/inference.py

```diff
@@ -24,6 +24,7 @@
 
 from . import constants as C
 from . import data_io
+from .lexicon import TopKLexicon
```

Comment:
could you do `from . import lexicon` and `lexicon.TopKLexicon` instead to be consistent with the rest of the code?

Reply:
Fixed.